src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -15,9 +15,16 @@
     //"OrchardCore_ContentLocalization_CulturePickerOptions": {
     //  "CookieLifeTime": 14 // Set the culture picker cookie life time (in days).
     //},
+    // See https://docs.orchardcore.net/en/latest/docs/reference/core/Data/#sqlite.
     //"OrchardCore_Data_Sqlite": {
     //  "UseConnectionPooling": false
     //},
+    // See https://docs.orchardcore.net/en/latest/docs/reference/core/Data/#database-table to configure database table presets used before a given tenant is setup.
+    //"OrchardCore_Data_DatabaseTable": {
```

Comment:
I suggest we name this `OrchardCore_Data_TableOptions` instead, Also I suggest put the default value in the commented out seeing like we do in everything else so something like this instead 
```
//"OrchardCore_Data_TableOptions": {
    //  "DefaultDocumentTable": "Document", // Document table name, defaults to 'Document'.
    //  "DefaultTableNameSeparator": "_", // Table name separator, one or multiple '_', "NULL" means no separator, defaults to '_'.
    //  "DefaultIdentityColumnSize": "Int64" // Identity column size, 'Int32' or 'Int64', defaults to 'Int64'.
    //},
```

Reply:
Okay